share/shwap/p2p/bitswap/sample_block.go

```diff
@@ -96,19 +96,28 @@ func (sb *SampleBlock) Populate(ctx context.Context, eds eds.Accessor) error {
 }
 
 func (sb *SampleBlock) UnmarshalFn(root *share.AxisRoots) UnmarshalFn {
-	return func(data []byte) error {
+	return func(cntrData, idData []byte) error {
 		if !sb.Container.IsEmpty() {
```

Comment:
Why it should be empty before unmarshaling? We don't want to overwrite the current data or what?

Reply:
Yes, there are possible edge cases in tests where this function is called twice for the same data

Reply:
Is this really a problem?